R/importNGWMN_wml2.R

```diff
@@ -147,3 +111,68 @@ importNGWMN_wml2 <- function(input, asDateTime=FALSE, tz=""){
   }
   return(mergedDF)
 }
+
+#' parse the timeseries portion of a waterML2 file
+#' 
+#' Returns data frame columns of all information with each time series measurement;
+#' Anything defined as a default, is returned as an attribute of that data frame.
+#' 
+#' @param input XML with only the wml2:MeasurementTimeseries node and children
+#' @importFrom xml2 xml_attr xml_find_all xml_text 
+#' @importFrom dplyr mutate
+#' @importFrom lubridate parse_date_time
+#' @export
+parseWaterML2Timeseries <- function(input, asDateTime) {
```

Comment:
Yeah I was considering that, I kept it here since this is the function that will probably use it the most, since nothing else uses ML2 by default.  I suppose conceptually it fits better with the original though.